### PR TITLE
happy thoughts! jonna

### PR DESCRIPTION
**Pull request by jonnaru**
**Link to my deployed project:**
https://happy-smiley-thoughts.netlify.app/

**My reflections on how this week's project turned out:**
Fun but confusing 😬

**Things I'd like to have clarified or explained in more detail:**
It has been a bit confusing with different ways to write the code. Just having components makes the brain explode so it was hard to understand what should be written where. Maybe be more clear how to solve a problem in one way instead of to many ways? 🤷‍♀️

### CODE REVIEW

File: code/src/assets/no-happy2.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...,75C335.58-23.5,173.5-21.13,75,80.29-21.47,179.63-21.47,337.67,75,437c101.42,98.5,263.5,96.13,362-5.29C533.47,332.37,533.47,174.33,437,75ZM415.81,415.81c-88.26,88.26-231.36,88.26-319.62,0s-88.26-231.36,0-319.62,231.36-88.26,319.62,0S504.07,327.55,415.81,415.81Z" transform="translate(-2.65 -2.65)"/><path d="M165,248.35a51.64,51.64,0,1,0-51.64-51.64A51.64,51.64,0,0,0,165,248.35Zm0-73.28a21.64,21.64,0,1,1-21.64,21.64h0A21.68,21.68,0,0,1,165,175.07Z" transform="translate(-2.65 -2.65)"/><path d="M345.37,248.35a51.64,51.64,0,1,0-51.64-51.64A51.64,51.64,0,0,0,345.37,248.35Zm0-73.28a21.64,21.64,0,1,1-21.64,21.64A21.64,21.64,0,0,1,345.37,175.07Z" transform="translate(-2.65 -2.65)"/><path d="M342.7,281.37H169.32a67.78,67.78,0,0,0-67.71,67.71h0a67.78,67.78,0,0,0,67.71,67.71H342.7a67.71,67.71,0,0,0,.07-135.42Zm34.58,52.71H340.19V311.37h2.51A37.77,37.77,0,0,1,377.28,334.08Zm-180.08,30h41.49v22.71H197.2Zm0-52.7h41.49v22.7H197.2Zm71.5,52.7h41.49v22.71H268.7Zm0-30V311.37h41.49v22.71ZM167.2,311.44v22.64H134.73A37.79,37.79,0,0,1,167.2,311.44Zm-32.47,52.64H167.2v22.64a37.75,37.75,0,0,1-32.47-22.64Zm208,22.71h-2.51V364.08h37.09a37.77,37.77,0,0,1-34.61,22.71Z" transform="translate(-2.65 -2.65)"/></svg>
```
Comment (Katsiez) [inline code/src/assets/no-happy2.svg:1]: Aha! You imported each smiley as an svg and a separate component! Wow, so much work, I am impressed Jonna!

**Review by Katsiez — APPROVED** [review]
Dear Jonna, where should I even start! I applaud you for the cleanliness of your code, clever solutions and for daring to use Math.abs 🤣☺️ Your creativity shines through this project, and even though we all had a similar task, you have found a way to make it your own, which is so impressive to me. You inspire me and I am so glad to have gotten a chance to peek into your code this week. The only minor suggestion I have is to implement the empty space function mentioned below, although that is of course completely up to you. Great job, be proud of yourself. 🎉🌈🙌✨🎊💜

File: code/src/components/Loading.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import "./loading.css";
+
+const Loading = () => {
+  return (
+    <>
```
Comment (EmOlsz) [inline code/src/components/Loading.js:7]: No need for React fragment (`</>`) if you return only one element (in your case `svg`)

File: code/src/components/MessageInput.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      headers: {
+        "Content-Type": "application/json",
+      },
+      // send the json as a string (object does not work here)
+      body: JSON.stringify({ message }), // = ({ message: message })
+    })
+      .then((res) => res.json())
+      .then((newMessage) => {
+        // adding new message to start of messages array
+        console.log(newMessage);
```
Comment (EmOlsz) [inline code/src/components/MessageInput.js:25]: Delete all console.logs before final push

**Pull request closed without merging**